src/constants.ts

```diff
@@ -57,6 +58,11 @@ export enum OperationType {
   upload = 'Upload device code'
 }
 
+export enum ScaffoldType {
```

Comment:
use EnvironmentType with value: local, container, wsl, remotemachine

Reply:
The type is not simply about the environment information.